.github/workflows/jeandle-llvm-test.yml

```
@@ -1,15 +1,30 @@
 name: "jeandle-llvm check"
 
 on:
-  pull_request_target:
-    branches:
-      - 'main'
+  push:
+    branches-ignore:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

If we trigger CI actions through PRs, we should write the config like this:
```suggestion
  group: ${{ github.workflow }}-${{ github.head_ref }}
```